1606577247681	Sync.SyncScheduler	DEBUG	Next sync in 3600000 ms. (why=schedule)
1606577338291	Sync.RemoteTabs	INFO	Generating tab list with filter
1606577338291	Sync.RemoteTabs	INFO	Final tab list has 0 clients with 0 tabs.
1606578621989	Sync.SyncScheduler	DEBUG	Network link status changed to "down". Offline?: false
1606578621989	Sync.SyncScheduler	DEBUG	Clearing sync triggers and the global score.
1606578621996	Sync.SyncScheduler	DEBUG	Next sync in 3600000 ms. (why=schedule)
1606578622010	Sync.SyncScheduler	DEBUG	Network link status changed to "down". Offline?: false
1606578622010	Sync.SyncScheduler	DEBUG	Clearing sync triggers and the global score.
1606578622517	Sync.SyncScheduler	DEBUG	Network link status changed to "up". Offline?: false
1606578622517	Sync.SyncScheduler	DEBUG	Network link looks up. Syncing.
1606578622519	Sync.SyncScheduler	DEBUG	Next sync in 3600000 ms. (why=schedule)
1606578622529	Sync.Service	DEBUG	User-Agent: Firefox/83.0 (Windows NT 10.0; Win64; x64) FxSync/1.85.0.20201112153044.desktop
1606578622529	Sync.Service	INFO	Starting sync at 2020-11-28 21:20:22 in browser session 2Wg3Sy1C9veE
1606578622530	Sync.SyncScheduler	DEBUG	Clearing sync triggers and the global score.
1606578622533	Sync.Status	INFO	Resetting Status.
1606578622533	Sync.Status	DEBUG	Status.service: success.status_ok => success.status_ok
1606578622534	FirefoxAccounts	DEBUG	FxAccountsProfileClient: Requested profile
1606578622534	FirefoxAccounts	DEBUG	getOAuthToken enter
1606578622537	FirefoxAccounts	TRACE	getCachedToken returning cached token
1606578622538	FirefoxAccounts	DEBUG	getOAuthToken returning a cached token
1606578622538	Services.Common.RESTRequest	DEBUG	GET request to https://profile.accounts.firefox.com/v1/profile
1606578622543	Sync.SyncScheduler	DEBUG	Network link status changed to "up". Offline?: false
1606578622543	Sync.SyncScheduler	DEBUG	Network link looks up. Syncing.
1606578622549	Sync.Service	DEBUG	User-Agent: Firefox/83.0 (Windows NT 10.0; Win64; x64) FxSync/1.85.0.20201112153044.desktop
1606578622549	Sync.Service	INFO	Starting sync at 2020-11-28 21:20:22 in browser session 2Wg3Sy1C9veE
1606578622549	Sync.Service	DEBUG	Exception calling anonymous function: LockException: Could not acquire lock. Label: "service.js: sync".(resource://services-sync/util.js:62:5) JS Stack trace: LockException@util.js:62:5
throwLockException@util.js:148:11
WrappedLock@util.js:162:15
_lockedSync@service.js:1370:6
sync/<@service.js:1334:18
WrappedCatch@util.js:133:27
sync@service.js:1335:7
1606578622550	Sync.Service	INFO	Cannot start sync: already syncing?
1606578623523	Sync.SyncScheduler	DEBUG	Network link status changed to "changed". Offline?: false
1606578623616	Services.Common.RESTRequest	DEBUG	GET https://profile.accounts.firefox.com/v1/profile failed: 2152398878 - NS_ERROR_UNKNOWN_HOST
1606578623616	FirefoxAccounts	ERROR	Background refresh of profile failed: {"name":"FxAccountsProfileClientError","code":null,"errno":998,"error":"NETWORK_ERROR","message":"[Exception... \"NS_ERROR_UNKNOWN_HOST\"  nsresult: \"0x804b001e (NS_ERROR_UNKNOWN_HOST)\"  location: \"JS frame :: resource://services-common/rest.js :: onStopRequest :: line 488\"  data: no]"}
1606578623628	Sync.Resource	WARN	GET request to https://sync-1-us-west1-g.sync.services.mozilla.com/1.5/174343456/info/collections?v=1.85.0 failed: [Exception... "The lookup of the hostname failed"  nsresult: "0x804b001e (NS_ERROR_UNKNOWN_HOST)"  location: "<unknown>"  data: no] No traceback available
1606578623628	Sync.Status	DEBUG	Status.sync: success.sync => error.login.reason.network
1606578623628	Sync.Status	DEBUG	Status.service: success.status_ok => error.sync.failed
1606578623628	Sync.ErrorHandler	ERROR	Sync encountered an error: [Exception... "The lookup of the hostname failed"  nsresult: "0x804b001e (NS_ERROR_UNKNOWN_HOST)"  location: "<unknown>"  data: no] No traceback available
1606578623628	Sync.SyncScheduler	DEBUG	Next sync in 3600000 ms. (why=reschedule)
1606578623630	Sync.Service	DEBUG	Exception calling anonymous function: [Exception... "The lookup of the hostname failed"  nsresult: "0x804b001e (NS_ERROR_UNKNOWN_HOST)"  location: "<unknown>"  data: no] No traceback available
